nightly.Jenkinsfile

```diff
@@ -204,6 +204,68 @@ stage('Build') {
 			}
 		})
 	}
+	// Don't run additional checks when this is a PR
```

Comment:
We run the nightly on PRs? :thinking:

Reply:
I hope not.. but then there's this

https://github.com/hibernate/hibernate-orm/blob/e407c53c861a0e9dc745a78b15a5b1afd9ffd7c7/nightly.Jenkinsfile#L77-L80

so 🤷🏻 I thought I'd better put that if